bluesky/simple_scans.py

```diff
@@ -1,506 +0,0 @@
-"""
-These "plans" bundle a Message generator with an instance of the RunEngine,
-combining two separate concepts -- instructions and execution -- into one
-object. This makes the interface less flexible and somewhat less "Pythonic"
-but more condensed.
-
-This module is meant to be run in a namespace where several global
-variables have been defined. If some variables are left undefined, the
-associated plans will be not usable.
-
-    DETS  # list of detectors
-    MASTER_DET  # detector to use for tw
-    MASTER_DET_FIELD  # detector field to use for tw
-    H_MOTOR
-    K_MOTOR
-    L_MOTOR
-    TH_MOTOR
-    TTH_MOTOR
-    TEMP_CONTROLLER
-
-Page numbers in the code comments refer to the SPEC manual at
-http://www.certif.com/downloads/css_docs/spec_man.pdf
-"""
-from inspect import signature
-import matplotlib.pyplot as plt
-from bluesky import plans
-from bluesky.callbacks import LiveTable, LivePlot, LiveRaster
-from bluesky.scientific_callbacks import PeakStats
-from boltons.iterutils import chunked
-from bluesky.global_state import gs
-from bluesky.utils import (normalize_subs_input, Subs, DefaultSubs,
-                           first_key_heuristic)
-from collections import defaultdict
-from itertools import filterfalse, chain, count
-
-# ## Factory functions acting a shim between plans and callbacks ###
-
-
-def table_from_motors(scan):
-    "Setup a LiveTable by inspecting a scan and gs."
-    # > 1 motor
-    return LiveTable(list(scan.motors) + gs.TABLE_COLS + [gs.PLOT_Y])
-
-
-def table_from_motor(scan):
-    "Setup a LiveTable by inspecting a scan and gs."
-    # 1 motor
-    return LiveTable([scan.motor] + gs.TABLE_COLS + [gs.PLOT_Y])
-
-
-def table_gs_only(scan):
-    "Setup a LiveTable by inspecting a scan and gs."
-    # no motors
-    return LiveTable(gs.TABLE_COLS + [gs.PLOT_Y])
-
-
-def _figure_name(base_name):
```

Comment:
Don't worry, @tacaswell: this code (and related functions below) were not deleted; they were just moved. This file `simple_scans.py`, home of "bundled scans," is no longer needed.